src/cases/containers/case-details/case-details.component.spec.ts

```diff
@@ -1,37 +1,38 @@
+import { HttpClientTestingModule } from '@angular/common/http/testing';
 import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { RouterTestingModule } from '@angular/router/testing';
 import {
-  CaseUIToolkitModule,
-  DraftService,
+  AbstractAppConfig,
   AlertService,
-  HttpService,
   AuthService as CCDAuthService,
-  CasesService,
-  HttpErrorService,
-  AbstractAppConfig,
   CaseEditWizardGuard,
-  RouterHelperService,
+  CasesService,
+  CaseUIToolkitModule,
   DocumentManagementService,
+  DraftService,
+  HttpErrorService,
+  HttpService,
   PageValidationService,
   PlaceholderService,
-  SearchService,
   RequestOptionsBuilder,
+  RouterHelperService,
   SearchFiltersModule,
+  SearchService,
 } from '@hmcts/ccd-case-ui-toolkit';
-import { AppConfig } from '../../../app/services/ccd-config/ccd-case.config';
+import { combineReducers, StoreModule } from '@ngrx/store';
 import { ScrollToService } from '@nicky-lenaers/ngx-scroll-to';
-import { RouterTestingModule } from '@angular/router/testing';
-import {combineReducers, StoreModule} from '@ngrx/store';
-import { SharedModule } from '../../../app/shared/shared.module';
-import { AppConfigService } from '../../../app/services/config/configuration.services';
 
-import { reducers } from 'src/app/store';
+import { AppConfig } from '../../../app/services/ccd-config/ccd-case.config';
+import { AppConfigService } from '../../../app/services/config/configuration.services';
+import { SharedModule } from '../../../app/shared/shared.module';
+import { reducers } from '../../../app/store';
 import * as fromCases from '../../store/reducers';
-import {CaseDetailsComponent} from './case-details.component';
-import { HttpClientTestingModule } from '@angular/common/http/testing';
+import { CaseDetailsComponent } from './case-details.component';
+
 class MockSortService {
-  features = {};
-  getFeatureToggle() { }
-  getEditorConfiguration() { }
+  public features = {};
+  public getFeatureToggle() { }
+  public cgetEditorConfiguration() { }
 }
 xdescribe('CaseDetailsComponent', () => {
```

Comment:
What is the point of this spec file if we have an xdescribe? Adnan mentioned on other PR that we should be getting rid of these so is this going to be implemented in future?

Reply:
@connorpgpmcelroy  its a good practice to remove it. But, not essential for this PR.